control_plane/src/pageserver.rs

```diff
@@ -247,7 +254,10 @@ impl PageServerNode {
     }
 
     fn pageserver_env_variables(&self) -> anyhow::Result<Vec<(String, String)>> {
```

Comment:
since this currently fail , shouldn't it return Result<Vec<(S,S)>, Infallible> ?

Reply:
It calls `generate_auth_token`, which can fail